alm/.env.example

```diff
@@ -1,15 +1,19 @@
-COMMON_HOME_BRIDGE_ADDRESS=0x2CA5411c4bf447Cc27CD6E6d1d046f922A27C399
-COMMON_FOREIGN_BRIDGE_ADDRESS=0x63C47c296B63bE888e9af376bd927C835014039f
+COMMON_HOME_BRIDGE_ADDRESS=0x1ee6E3E3d2DE779858728E157B3B9C488bA7b706
```

Comment:
can you leave this file untouched and open a new one like .env.bsc.example

Reply:
https://github.com/fuseio/tokenbridge/pull/7/commits/7092d5e8c1a54c80a395753c1a409dfc1e4faaf0